content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

```diff
@@ -104,10 +104,12 @@ be done by the egress gateway, as opposed to by the sidecar in the previous exam
       servers:
       - port:
           number: 80
-          name: http-port-for-tls-origination
-          protocol: HTTP
+          name: https-port-for-tls-origination
+          protocol: HTTPS
```

Comment:
does HTTPS even work on port 80?

Reply:
@nschhina from a conceptual networking standpoint or from Istio configuration point of view? There are almost no Egress Gateway listeners by default as they need to be explicitly defined by a Gateway resource, so creating a HTTPS listener on port 80 will not interfere with any default port 80 HTTP traffic on the egress gateway as there is none by default.

Reply:
I see, just don't think this is a good practice to listen for HTTPS on standard HTTP port even if there's no conflicts. Wouldn't the PR istio/istio#25588 make these changes you made useless anyways? Maybe waiting for that bug fixed to get merged is better

Reply:
istio/istio#25588 is a bug fix so communication between sidecar's and egress gateway can communicate securely with manually configured Istio MTLS.  It will not resolve the problems that exist in the docs without the changes provided here - both are required, not one or the other.  Auto MTLS broke the security aspect because auto mtls traffic between egress and proxy is not configured by default.  SDS changes broke additional parts because the certs were physically removed from the pods and placed in k8s secrets instead

Egress Gateway TLS Origination steps work as is on istio.io, but traffic is not secure - so we need to provide an example for this since it is required for most customers

Egress Gateway with MTLS Origination steps do not work right now and as is, and need istio/istio#25588 in addition to the changes I have made to the snips

Reply:
Got it, can you also enable the last check in mtls_test.sh as well so we know for sure nothing is broken after istio/istio dep is updated with that fix. As for simple TLS example it 100% needs to be updated

Reply:
Is there a valid reason why the example still does https over port 80, even as of today? As @nschhina also noted I don't think it's a good practice to listen for HTTPS on port 80.

On a side note, this is a bit scary:
> Auto MTLS broke the security aspect because auto mtls traffic between egress and proxy is not configured by default. SDS changes broke additional parts because the certs were physically removed from the pods and placed in k8s secrets instead

Have those issues been resolved?